cluster_management/src/main/java/com/pinterest/rocksplicator/Utils.java

```diff
@@ -177,6 +177,11 @@ public static void logTransitionMessage(Message message) {
         + " for " + message.getPartitionName());
   }
 
+  public static void logTransitionCompletionMessage(Message message) {
+    LOG.error("Switched from " + message.getFromState() + " to " + message.getToState()
```

Comment:
I feel like it would be difficult to read the difference between "Switching" and "Switched." Maybe change this to `"Completed switch from "`?

Reply:
Agreed:

Replaced with 

Transition Started
Transition Completed